diff --git a/CMakeLists.txt b/CMakeLists.txt
index a0318b9..d89ed98 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -19,6 +19,8 @@ endif()
 message (STATUS "Installation path will be ${CMAKE_INSTALL_PREFIX}")
 include(GNUInstallDirs)
 
+set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
+
 if(PYSTRING_HEADER_ONLY)
     message(STATUS "Building pystring as header-only library")
     add_library(pystring INTERFACE)
@@ -48,9 +50,9 @@ else()
     )
 
     install(TARGETS pystring
-            LIBRARY DESTINATION lib
-            RUNTIME DESTINATION bin
-            ARCHIVE DESTINATION lib
+        RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}   # pystring.dll (Windows)
+        LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}   # .so / .dylib
+        ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}   # pystring.lib / static
     )
 
     install (FILES pystring.h
