diff --git a/CMakeLists.txt b/CMakeLists.txt
index a0318b9..b1d8511 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -48,9 +48,9 @@ else()
     )
 
     install(TARGETS pystring
-            LIBRARY DESTINATION lib
-            RUNTIME DESTINATION bin
-            ARCHIVE DESTINATION lib
+        RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}   # pystring.dll (Windows)
+        LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}   # .so / .dylib
+        ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}   # pystring.lib / static
     )
 
     install (FILES pystring.h
